plugins/hamma_plot.py

```diff
@@ -0,0 +1,121 @@
+"""
+Plugin to generate a plot of HAMMA data and save it locally.
+"""
+
+# Third party imports
+import matplotlib  # Do this first to ensure backend
+matplotlib.use('Agg')
+from matplotlib.pyplot import close as mpl_close
+from matplotlib.pyplot import subplots as mpl_subplots
+
+import hamma
+from hamma.plotting import plot as hamma_plot
+
+# Local imports
+import brokkr.pipeline.decode
+from brokkr.config.unit import UNIT_CONFIG
+from brokkr.config.metadata import METADATA
+
+# Standard library imports
+from pathlib import Path
+
+
+class HammaPlot(brokkr.pipeline.base.OutputStep):
+    """Generate a plot of HAMMA data."""
+
+    def __init__(self,
+                 min_update_time,
+                 save_path,
+                 **output_step_kwargs):
+        """
+        Make a plot of HAMMA data.
+
+        This step will "inject" a new entry into the dictionary Brokkr passes
+        in pipelines that tracks the last time a plot was made.
+
+        Parameters
+        ----------
+        min_update_time : numeric
+            The minimum time (in seconds) that should elapse before a new plot
+            is saved.
+        save_path : str
+            The path to which the plot is saved.
+        output_step_kwargs : **kwargs, optional
+            Keyword arguments to pass to the OutputStep constructor.
+
+        """
+
+        self._previous_data = None
```

Comment:
```suggestion
        self._last_save_time = None
```

Just make this a simple instance attribute